integrations/weaviate/src/haystack_integrations/document_stores/weaviate/document_store.py

```diff
@@ -294,6 +292,16 @@ def _to_data_object(self, document: Document) -> Dict[str, Any]:
                 "Document %s has the unsupported `_split_overlap` meta field. It will be ignored.", data["_original_id"]
             )
 
+        if "dataframe" in data:
+            dataframe = data.pop("dataframe")
+            if dataframe:
+                logger.warning(
+                    "Document %s has the `dataframe` field set. "
+                    "WeaviateDocumentStore no longer supports dataframes and this field will be ignored. "
+                    "The `dataframe` field will soon be removed from Haystack Document.",
+                    data["_original_id"],
+                )
```

Comment:
I'd double check that this warning message is formatted correctly. I believe we have to format the string like 
```
        logger.debug(
            "Connecting '{sender_component}.{sender_socket_name}' to '{receiver_component}.{receiver_socket_name}'",
            sender_component=sender_component_name,
            sender_socket_name=sender_socket.name,
            receiver_component=receiver_component_name,
            receiver_socket_name=receiver_socket.name,
        )
```
I'm unsure if the `%s` syntax also works

Reply:
yes, fixed now.